README.md

```diff
@@ -80,6 +80,104 @@ See [src/examples/](src/examples/)  sub-directory for more infos
 c++ -o program -I/path/to/highfive/include source.cpp  -lhdf5
 ```
 
+#### H5Easy
+
+For several 'standard' use cases the [HighFive/H5Easy.hpp](include/HighFive/H5Easy.hpp) interface is available. It allows:
+
+*   Reading/writing in a single line of:
+
+    -   scalars (to/from an extendible DataSet),
+    -   strings,
+    -   vectors (of standard types),
+    -   [Eigen::Matrix](http://eigen.tuxfamily.org) (optional, enabled by including Eigen before including HighFive, or by `#define HIGHFIVE_EIGEN` before including HighFive),
+    -   [xt::xarray](https://github.com/QuantStack/xtensor) and [xt::xtensor](https://github.com/QuantStack/xtensor) (optional, enabled by including xtensor before including HighFive, or by `#define HIGHFIVE_XTENSOR` before including HighFive).
+
+*   Direct access to the `size` and `shape` of a DataSet.
+
+Consider this example:
+
+```cpp
+#include <Eigen/Eigen>
```

Comment:
Nice example, but it should be in a dedicated file in `src/examples/`, executed by CI instead.

Reply:
Agreed. README.md should be a minimal introduction.